examples/github-complete/main.tf

```
@@ -81,6 +81,8 @@ module "atlantis" {
   alb_log_bucket_name             = module.atlantis_access_log_bucket.this_s3_bucket_id
   alb_log_location_prefix         = "atlantis-alb"
   alb_listener_ssl_policy_default = "ELBSecurityPolicy-TLS-1-2-2017-01"
+  alb_enable_deletion_protection  = var.alb_enable_deletion_protection
+  alb_drop_invalid_header_fields  = var.alb_drop_invalid_header_fields
```

lets set this line to `true`:
```suggestion
  alb_drop_invalid_header_fields  = true
```